tests/compiletest.rs

```diff
@@ -61,6 +63,16 @@ fn run_tests(mode: Mode, path: &str, target: Option<String>) -> Result<()> {
         path_filter: path_filter.collect(),
         program: miri_path(),
         output_conflict_handling,
+        manifest_path: use_std
```

Comment:
"manifest_path" is not a very clear name (manifest of what).
`dependency_crate_manifest_path` might be a verbose way of stating this, or `dep_crate_manifest` as a shorter version.